scipy/linalg/_generate_pyx.py

```diff
@@ -689,7 +727,8 @@ def make_all(blas_signature_file="cython_blas_signatures.txt",
 
     comments = ["This file was generated by _generate_pyx.py.\n",
                 "Do not edit this file directly.\n"]
-    ccomment = ''.join(['/* ' + line.rstrip() + ' */\n' for line in comments]) + '\n'
+    ccomment = ''.join(['/* ' + line.rstrip() + ' */\n'
+                        for line in comments]) + '\n'
```

Comment:
This produces a double newline after the last element. Not sure if that's intended.

Reply:
Yah, that's not needed. I'll remove it.